pod/perldelta.pod

```diff
@@ -45,6 +45,17 @@ XXX For a release on a stable branch, this section aspires to be:
 
 [ List each incompatible change as a =head2 entry ]
 
+=head2 Apostrophe is no longer recognized as a global name separator
+
+The apostrophe C<'> has been recognized as a separator between
+components of package names and package variables since Perl 4, acting
+exactly like C<::> which was introduced in Perl 5.
+
+Use of C<'> as a package separator was deprecated in Perl 5.38, has
```

Comment:
This feels like a comma splice. I'd either replace the comma by "and" or use separate sentences ("… in Perl 5.38. It has …").

Reply:
thanks, fixed.